helpers/handlebarsHelpers.js

```diff
@@ -0,0 +1,96 @@
+const Handlebars = require('handlebars');
+const config = require('config');
+
+module.exports = {
+    // define any extra helpers you may need
+  googleAnalytics () {
+    const googleApiKey = config.get('AnalyticsConfig.GoogleId');
+    return new Handlebars.SafeString(
+            `<script>
+        (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+        (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
```

Comment:
Why do these have to be in a helper instead of just in a normal template?

(I know nothing about Handlebars -- you can just say it has to be this way if that's the case :) )

Reply:
They have to be in a helper so that they can be generated based on the proper google analytics key from the environmental vars